src/assessments/types/report-instance-field.ts

```diff
@@ -14,6 +14,7 @@ export type ReportInstanceFieldMap = { [KEY in string]?: ReportInstanceField };
 export type ReportInstanceFields = ReportInstanceField[];
 
 type HasPropertyBag<PB> = { propertyBag: PB };
+type PropertyBagKey<PB> = string | PB[keyof PB & string];
```

Comment:
this looks not to strong of a type... if the value is a `string` then it can be any string right? or am I missing something?

Reply:
The way I am normally assigning types is to check what the function is normally returning as of now and then just add it like it.

Reply:
Does it look okay now?